src/policies/generic.ts

```diff
@@ -8,13 +8,28 @@ interface Permission {
 interface User {
   permissions: Permission[];
   roles?: { name: string }[];
+  flags?: string[];
+}
+
+interface DBUser {
+  get: (options?: { plain: boolean }) => User;
 }
 
 export default class Generic {
```

Comment:
this file feels a bit like a rewrite of the users policy, we probably at some point want to condense as they are getting at the same functionality

Reply:
They started diffrent, but turned into the same thing. I will try to move over the the users one.